docs/bugs_and_glitches.md

```diff
@@ -14,7 +14,7 @@ Some fixes are mentioned as breaking compatibility with link battles. This can b
 
 
 ## Contents
-
+- [Berserk Gene's confusion isn't permanent](#berserk-genes-confusion-isnt-permanent)
```

Comment:
Keep a blank line between the heading and the list. Also, if this is only considered a bug due to Stadium 2 changing it, the bug should be that it lasts for 256 turns and not the standard duration.

Also, this seems like a more unambiguous bug: "If a confused Pokémon is switched out and replaced by a Pokémon holding a Berserk Gene, the holder will be confused for the remaining duration of the first Pokémon's confusion, rather than for 256 turns."

Reply:
Fixed in the latest commit.